components/eamxx/src/physics/ace/CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+set(ACE_SRCS
+  eamxx_ace_process_interface.cpp
+  ace_grids_manager.cpp
+)
+
+# Initialize PyTorch found status to FALSE
+set(SCREAM_FOUND_PYTORCH FALSE CACHE BOOL "Whether PyTorch was found" FORCE)
+
+# Use a configurable path for PyTorch with a default value
+if(NOT DEFINED TORCH_CMAKE_PATH)
+  if (Kokkos_ENABLE_CUDA)
+    set(GPU_OR_CPU "" CACHE STRING "GPU or CPU version of PyTorch.")
+    set(TORCH_CMAKE_PATH "/global/cfs/cdirs/e3sm/software/libtorch/libtorch-cxx11-abi-shared-with-deps-2.6.0+cu124" 
+        CACHE PATH "Path to PyTorch CMake files")
+    message(STATUS "Will try to use GPU version of PyTorch at ${TORCH_CMAKE_PATH} with ${GPU_OR_CPU}")
+  else()
+    set(TORCH_CMAKE_PATH "/global/cfs/cdirs/e3sm/software/libtorch/libtorch-cxx11-abi-shared-with-deps-2.6.0+cpu" 
+        CACHE PATH "Path to PyTorch CMake files")
+    message(STATUS "Will try to use CPU version of PyTorch at ${TORCH_CMAKE_PATH} with ${GPU_OR_CPU}")
+  endif()
+  # TODO add support for ROCm
+endif()
+
+# Try to find PyTorch, but don't require it
+set(CMAKE_PREFIX_PATH ${TORCH_CMAKE_PATH})
+find_package(Torch QUIET)
```

Comment:
Why not adding `REQUIRED` so we error out if it doesn't find it (and do away with the if conditional below)? I don't think it makes sense to continue the build without ACE if the user explicitly requested the ACE compset...

Reply:
Are you still planning on making torch REQUIRED when EAMXX_ENABLE_ACE=ON?